### ISSUE TITLE
Lot's of errors when adding a CuboidCollider to the scene

### ISSUE THREAD

**Issue by Seanmclem** [ErdongChen-Andrew/CharacterControl#7]
I get Lot's of errors when adding a CuboidCollider to the scene. Just from adding it

like

```
Uncaught TypeError: Cannot read properties of undefined (reading 'x')
    at Object.current (CharacterController.jsx:667:27)
```

And 

```
Uncaught TypeError: Cannot read properties of null (reading 'bodyType')
    at Object.current (CharacterController.jsx:609:59)
```

Edit: Code sandbox in later comment below

**Comment by ErdongChen-Andrew** [comment]
Hello @Seanmclem , I'm unable to access the sandbox as I'm not a member of the team. Could you please provide more details about the issue? Where were you trying to add the CuboidCollider?

**Comment by Seanmclem** [comment]
> Hello @Seanmclem , I'm unable to access the sandbox as I'm not a member of the team. Could you please provide more details about the issue? Where were you trying to add the CuboidCollider?

Maybe [this link will work](https://codesandbox.io/p/github/Seanmclem/CharacterControlBug/main?file=%2Fsrc%2FExperience.jsx%3A69%2C12&layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522clm6frkze000j3t6bcpk8pfkq%2522%252C%2522sizes%2522%253A%255B70%252C30%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522clm6frkze000e3t6ba2mdv1xs%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522clm6frkze000i3t6bh0wmj9ti%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522clm6frkze000g3t6b6q08beh6%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%252C%2522sizes%2522%253A%255B50%252C50%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522clm6frkze000e3t6ba2mdv1xs%2522%253A%257B%2522id%2522%253A%2522clm6frkze000e3t6ba2mdv1xs%2522%252C%2522activeTabId%2522%253A%2522clm6fs06d011b3t6biv58b3ys%2522%252C%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clm6frkze000d3t6b8dtt990x%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522filepath%2522%253A%2522%252F.DS_Store%2522%252C%2522state%2522%253A%2522IDLE%2522%257D%252C%257B%2522type%2522%253A%2522FILE%2522%252C%2522filepath%2522%253A%2522%252Fsrc%252Findex.jsx%2522%252C%2522id%2522%253A%2522clm6frvdn00st3t6bhqdmtmrx%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522state%2522%253A%2522IDLE%2522%257D%252C%257B%2522type%2522%253A%2522FILE%2522%252C%2522filepath%2522%253A%2522%252Fsrc%252FExperience.jsx%2522%252C%2522id%2522%253A%2522clm6fs06d011b3t6biv58b3ys%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522state%2522%253A%2522IDLE%2522%257D%255D%257D%252C%2522clm6frkze000g3t6b6q08beh6%2522%253A%257B%2522id%2522%253A%2522clm6frkze000g3t6b6q08beh6%2522%252C%2522activeTabId%2522%253A%2522clm6frtin00o73t6bsew6tuag%2522%252C%2522tabs%2522%253A%255B%257B%2522type%2522%253A%2522TASK_PORT%2522%252C%2522taskId%2522%253A%2522dev%2522%252C%2522port%2522%253A5173%252C%2522id%2522%253A%2522clm6frtin00o73t6bsew6tuag%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522path%2522%253A%2522%252F%2522%257D%255D%257D%252C%2522clm6frkze000i3t6bh0wmj9ti%2522%253A%257B%2522id%2522%253A%2522clm6frkze000i3t6bh0wmj9ti%2522%252C%2522activeTabId%2522%253A%2522clm6frrsa00j43t6by0b118s1%2522%252C%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clm6frkze000h3t6b2iihkz4l%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522TERMINAL%2522%252C%2522shellId%2522%253A%2522clm6frl3q001ae5kk7ui41doi%2522%257D%252C%257B%2522type%2522%253A%2522TASK_LOG%2522%252C%2522taskId%2522%253A%2522dev%2522%252C%2522id%2522%253A%2522clm6frrsa00j43t6by0b118s1%2522%252C%2522mode%2522%253A%2522permanent%2522%257D%255D%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Atrue%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D)? Seems to. 

Basically just dropped one in the Experience component. Just to start testing around with it, but immediately getting repeated errors and scene never fully loads

just literally adding 

```
         <CuboidCollider
          args={[15, 10, 15]}
          sensor
          onIntersectionEnter={() => console.log("Goal!")}
        />
```

Edit: added link

**Comment by ErdongChen-Andrew** [comment]
I see! Currently, the controller does not support the sensor body type. When the character and camera are inside a collider, it triggers an error in every frame. I'll address this issue in the next update. Thanks for bringing this up! 👍

**Comment by ErdongChen-Andrew** [comment]
Hello @Seanmclem , I believe the bug has been fixed. Let me know if you encounter any further issues.

**Comment by Seanmclem** [comment]
@ErdongChen-Andrew thanks I'll check it out and close the issue
